lib/lightning/workflows.ex

```diff
@@ -463,40 +463,12 @@ defmodule Lightning.Workflows do
           "coalesce(?, ?)",
           t.custom_path,
           type(t.id, :string)
-        ) == ^path,
+        ) == ^path and t.type == :webhook,
       preload: ^preloads
     )
     |> Repo.one()
   end
 
-  @doc """
-  Gets a single `Trigger` by its `custom_path` or `id`.
-
-  ## Parameters
-  - `path`: A binary string representing the `custom_path` or `id` of the trigger.
-
-  ## Returns
-  - Returns a `Trigger` struct if a trigger is found.
-  - Returns `nil` if no trigger is found for the given `path`.
-
-  ## Examples
-
-  ```
-  Lightning.Workflows.get_trigger_by_webhook("some_path_or_id")
-  # => %Trigger{id: 1, custom_path: "some_path_or_id", ...}
-
-  Lightning.Workflows.get_trigger_by_webhook("non_existent_path_or_id")
-  # => nil
-  ```
-  """
-  def get_trigger_by_webhook(path) when is_binary(path) do
```

Comment:
this was all unused code because of https://github.com/OpenFn/lightning/commit/c6f34403e1cbae8b6e4f9d6d9cbea51dfc94637a#r162886249 - i've looked at the two and don't think there's much difference